src/utils.jl

```diff
@@ -22,3 +22,8 @@ function invert_dims(A::AbstractArray, dims)
 
     return inverted_dims
 end
+
+_to_vec(x::AbstractArray) = x
+_to_vec(x::Tuple) = x
+_to_vec(x::Nothing) = x
+_to_vec(x) = [x]
```

Comment:
You may want to cross-reference `FiniteDifferences.to_vec`.
Which if you need that behavour we could split into its own package

Reply:
Thanks, definitely something to keep in mind. I think this is a simple and special case that should live on its own for now. In fact calling it `_to_vec` may just be confusing. Based on its current use a better name might be `_convert_cols`.